features/step_definitions/controllers-steps.js

```diff
@@ -207,18 +207,23 @@ Then('I send a HTTP {string} request with:', async function (method, dataTable)
     method,
     body: {
       jwt: this.sdk.jwt,
-      ...body,
+      ...body
```

Comment:
It's nitpicking but the comma should actually stays here.  
It makes refactor more easier because you can cut/copy + paste any line anywhere